doozer/doozerlib/cli/release_gen_payload.py

```diff
@@ -1893,3 +1933,20 @@ async def check_nightlies_consistency(assembly_inspector: AssemblyInspector) ->
         results = await asyncio.gather(*tasks)
         issues.extend([issue for result in results for issue in result])
         return issues
+
+    @staticmethod
+    def does_payload_entry_include_embargo(entries) -> List[AssemblyIssue]:
```

Comment:
sounds like a boolean, should describe what it's returning
```suggestion
    def embargo_issues_for_payload(entries) -> List[AssemblyIssue]:
```